src/Makefile

```diff
@@ -39,9 +39,10 @@ $(obj)/%.o: $(src)/%.c GET_STACK_SIZE FORCE
 	$(call if_changed_rule,cc_o_c)
 	$(OBJCOPY) $(OBJCOPYFLAGS) $(OBJCOPYFLAGS_$(@F)) $@
 
-scheduler-objs := $(objs-y) main.o tainted.o sched_rebuild.o
+scheduler-objs := $(objs-y) main.o bypass_test.o tainted.o sched_rebuild.o
```

Comment:
Don't put test code in src !!! If you want to such kind of test, please try other ways.

Reply:
OK I'll drop this patch